tests/tools/test_ontodoc.py

```diff
@@ -0,0 +1,15 @@
+"""Test the `ontodoc` tool."""
+from pathlib import Path
+
+import pytest
+
+
+@pytest.mark.skip("ontodoc is tested in other ways")
+@pytest.mark.parametrize("tool", ["ontodoc"], indirect=True)
+def test_run(tool, tmpdir: Path) -> None:
+    """Check running `ontodoc` works."""
+    test_file = (
+        Path(__file__).resolve().parent.parent / "testonto" / "models.ttl"
+    )
+
+    tool.main([str(test_file), str(tmpdir / "test.ttl")])
```

Comment:
The output of ontodoc should be something supported by pandoc. 
```suggestion
    tool.main([str(test_file), str(tmpdir / "test.md")])
```